### ISSUE TITLE
Linter

### ISSUE THREAD

**Issue by lordmauve** [lordmauve/pgzero#35]
Originally reported by: **Daniel Pope (Bitbucket: [lordmauve](https://bitbucket.org/lordmauve), GitHub: [lordmauve](https://github.com/lordmauve))**

----------------------------------------

We should provide a custom linger to detect common problems at edit time.

----------------------------------------
- Bitbucket: https://bitbucket.org/lordmauve/pgzero/issue/35

**Comment by pintman** [comment]
I agree to #146. It would be great to have support for autocompletion in IDEs. In some way this can be achieved with something like

    import pgzero.keyboard
    keyboard = pgzero.keyboard.keyboard

However the following lines fail at runtime (maybe related to #126):

    import pgzero.game
    screen = pgzero.game.screen
